.github/workflows/rss-runner.yaml

```
@@ -0,0 +1,33 @@
+name: Events RSS Feed Cron Job
+
+on:
+  schedule:
+    - cron: "0 30 14 * * ?"
```

actionlint (1.7.7)

invalid CRON format "0 30 14 * * ?" in schedule event: expected exactly 5 fields, found 6: [0 30 14 * * ?]

(events)